Werewolf for Telegram/Languages/English.xml

```diff
@@ -2253,4 +2253,7 @@
   <string key="ExitAnonymousMode">
     <value>You can't use this bot as anonymous admin! Please exit the anonymous mode (or ask the admin who promoted you to do so) and try again!</value>
   </string>
+  <string key="DoppelgängerEnd">
+    <value>Doppelgänger {0} is alone in the village! how is it possible? Well, it seems that he was his own role model and for this reason he did not become another role... Now he is the last survivor of the village and he will remain a Doppelgänger forever. What a fate!</value>
+  </string>
```

Comment:
Undo this.

Language files are not to be committed on GitHub but to be uploaded through Telegram (See https://t.me/werewolftranslation).
Also, the string could use some rewording - maybe ask a native speaker to help you.
Also, the strings regarding the DG role all have "Doppelganger" (not "Doppelgänger") in their keys so far, so you should stick to it.
Also, use gender neutral language ("their" instead of "his", and so on)
And ALSO, that a DG is among the last 4 alive players, doesn't mean they are their own role model. They could very well have the tanner/thief/sorcerer as their role model.